src/coll/options.rs

```diff
@@ -641,7 +696,7 @@ pub struct FindOptions {
 
     /// Tags the query with an arbitrary string to help trace the operation through the database
     /// profiler, currentOp and logs.
-    pub comment: Option<String>,
+    pub comment: Option<Bson>,
```

Comment:
Because this option is already in our public API, we need to leave it as a string. Otherwise, users who are already using the option will get compilation errors because they're no longer specifying the correct type. Was this change needed for the sake of test file deserialization?

(From looking at the specification for this option in the CRUD spec, the server does accept arbitrary BSON on 4.4+, but it doesn't seem like we've made that change yet in Rust.)